docs/specifications/tests/Connectivity-TP/connectivity01.md

```diff
@@ -1,39 +1,218 @@
-## CONNECTIVITY01: UDP connectivity
+# CONNECTIVITY01: UDP connectivity to name servers
```

Comment:
Test Case name change also required in the [README ](https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/README.md)

Reply:
Not needed. That part of README is generated by script at release, see https://github.com/zonemaster/zonemaster/tree/master/utils#generatetestcaselistpl